# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1377018677 -7200
#      Tue Aug 20 19:11:17 2013 +0200
# Node ID 756bb4b58ad0f2c1c807d196fcd170c68ecc2872
# Parent  17be601bc7833a50630d195e0ad27dd91b187bfb
* main-window.h: exclude editor from widget list when qscintilla is missing

diff --git a/libgui/src/main-window.h b/libgui/src/main-window.h
--- a/libgui/src/main-window.h
+++ b/libgui/src/main-window.h
@@ -267,17 +267,19 @@ private:
   workspace_view *workspace_window;
   QList<octave_dock_widget *> dock_widget_list ()
   {
     QList<octave_dock_widget *> list = QList<octave_dock_widget *> ();
     list.append (static_cast<octave_dock_widget *> (command_window));
     list.append (static_cast<octave_dock_widget *> (history_window));
     list.append (static_cast<octave_dock_widget *> (file_browser_window));
     list.append (static_cast<octave_dock_widget *> (doc_browser_window));
+#ifdef HAVE_QSCINTILLA
     list.append (static_cast<octave_dock_widget *> (editor_window));
+#endif
     list.append (static_cast<octave_dock_widget *> (workspace_window));
     return list;
   }
 
   QToolBar *_main_tool_bar;
   QMenu *_debug_menu;
 
   QAction *_debug_continue;
